pkg/qemu/qemu.go

```diff
@@ -134,9 +134,11 @@ func appendArgsIfNoConflict(args []string, k, v string) []string {
 		return append(args, k)
 	}
 
-	if origV, ok := argValue(args, k); ok {
-		logrus.Warnf("Not adding QEMU argument %q %q, as it conflicts with %q %q", k, v, k, origV)
-		return args
+	if k != "-global" {
```

Comment:
Why this check?

Reply:
Since we need to allow multiple -global arguments which doesn't seems to conflict according to UTM.

Reply:
Then you can use `args = append(args, "-global", ...)` without using `appendArgsIfNoConflict`